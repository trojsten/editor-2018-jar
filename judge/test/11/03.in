STR:
General
P
STR:
Plukovnik
KA
STR:
Podplukovnik
IZEVITT
STR:
Strateg
PYJWYB
STR:
Spion
POSDOBZ
STR:
Programator
VJTRJCS
STR:
Velmajstermaxisupergeneralveduci
MBNJBJGPUZ
STR:
Nadpodpredzadplukovnikazarovendokoncaajgeneral
ZYWIKEVH
