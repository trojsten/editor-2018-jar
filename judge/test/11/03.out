{"Velmajstermaxisupergeneralveduci": "ZYWIKEVH", "Spion": "VJTRJCS", "Nadpodpredzadplukovnikazarovendokoncaajgeneral": "MBNJBJGPUZ", "Programator": "KA", "General": "PYJWYB", "Strateg": "POSDOBZ", "Podplukovnik": "P", "Plukovnik": "IZEVITT"}